source/sessions/tests/dirty-session-errors.yml

```diff
@@ -35,10 +31,14 @@ tests:
           session: session0
       - name: endSession
         object: session0
-      - name: assertServerSessionPoolSize
-        object: testRunner
+      - &find_with_implicit_session
```

Comment:
This diff removes the `insertOne` operation above, such that the sequence of operations is:

 * assertSessionNotDirty
 * endSession
 * find_with_implicit_session
 * assertSameLsidOnLastTwoCommands

The APM assertions below still include an `insert` command, though. I think the original intention was to use an explicit session for the insert and then return it to the pool to be used by the find command.

Reply:
The insertOne operation is still there.